selenium_tests/lti_emailer/page_objects/emailer_base.py

```diff
@@ -0,0 +1,27 @@
+from selenium.common.exceptions import NoSuchElementException
```

Comment:
It would be nice to have a convention to how the files and classes are named. I like the icommons_lti_tools style:

emailer_base_page.py (`class EmailerBasePage(...)`)
emailer_main_page.py (`class EmailerMainPage(...)`)
...etc